packages/plugin-aws-sdk/test/testing-utils.ts

```diff
@@ -1,4 +1,5 @@
 import { isInstrumentationSuppressed, context } from '@opentelemetry/api';
+import expect from 'expect';
```

Comment:
why do we need it here, and do not need it in other test files?

Reply:
expect is included in jest, since we moved to mocha we need to import it.